src/main/java/com/google/devtools/build/lib/skyframe/LocalDiffAwareness.java

```diff
@@ -85,7 +87,7 @@ public Factory(ImmutableList<String> excludedNetworkFileSystemsPrefixes) {
 
     @Override
     @Nullable
-    public DiffAwareness maybeCreate(Root pathEntry) {
+    public DiffAwareness maybeCreate(Root pathEntry, ImmutableSet<com.google.devtools.build.lib.vfs.Path> ignorePaths) {
```

Comment:
nit: import.

Reply:
Happy to tweak but I'm not exactly sure what change to make here! Existing code like L91 references this as `com.google.devtools.build.lib.vfs.Path` since `java.nio.file.Path` is already imported.

Reply:
Ah, I missed that. Please ignore the previous comment then.